### ISSUE TITLE
Using `align: const Alignment(0.0, 0.0)` in `BotToast.showXXX()` methods does not put the toast content in the center of the main content.

### ISSUE THREAD

**Issue by robinzhangx** [MMMzq/bot_toast#156]
**Describe the bug**
Using `align: const Alignment(0.0, 0.0)` in `BotToast.showXXX()` methods does not put the toast content in the center of the main content.

**To Reproduce**
```
    BotToast.showCustomText(
      toastBuilder: (_) => Text("some text"),
      align: const Alignment(0.0, 0.0),
      duration: duration,
    );
```

**Version (please complete the following information):**
 - BotToast Version : [4.1.1]

**Comment by MMMzq** [comment]
This issue has been fixed in version 4.1.2.

**Comment by robinzhangx** [comment]
Thanks.
